libcaf_net/caf/net/web_socket/framing.cpp

```diff
@@ -10,6 +10,49 @@
 
 namespace caf::net::web_socket {
 
+// -- static utility functions -------------------------------------------------
+
+expected<std::pair<uint16_t, const_byte_span>>
+framing::decode_closing_payload(const_byte_span payload) {
+  if (payload.size() == 0)
+    return std::make_pair(0, payload);
+  if (payload.size() < 2)
+    return make_error(caf::sec::protocol_error,
+                      "no status flag in closing payload");
+  auto status = (std::to_integer<uint_fast16_t>(payload[0]) << 8)
```

Comment:
Auto-completion mistake? It's inconsistent with the line immediately below and also doesn't make intuitive sense to me.

```suggestion
  auto status = (std::to_integer<uint16_t>(payload[0]) << 8)
```

Reply:
Mistake, you're probably right about the autocomplete.